src/pages/hegel/guides/general-division-of-the-logic.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Division of the Logic
+description: Here is a quick overview of the Science of Logic
+---
+
+## General Division of the Logic
+
+### The Major Divisions
+
+```mermaid
+mindmap
+    root((The Science Of Logic))
+        The Objective Logic
+            📙 Doctrine of Essence
+            📘 Doctrine of Being
+        The Subjective Logic
+            📔 The Doctrine of the Concept
+```
+
+_The Science of Logic_ is divided into two major parts or three doctrines. In accordance with the major parts, the _Logic_ is divided into _objective logic_ and _subjective logic_. The objective logic contains the doctrines _The Doctrine of Being_ and _The Doctrine of Essence_, and the subjective logic contains _The Doctrine of the Concept_. Each of these doctrines deal with the development of logic in predominant descriptors, such as immediate existence, mediated or reflected determinations and conceptual thought proper. In other words, each of these doctrines operate in ways that distinguish what is logically understood whether the matter is fundamentally immediate, mediated or a developmental integration of both. 
```

Comment:
The objective logic contains the doctrines: - I would add a colon or a comma before you list the Doctrine of Being and of Essence